test/main.hs

```diff
@@ -24,8 +27,8 @@ main :: IO ()
 main =
   bracket (connect testConn) close $ \conn ->
     hspec $ do
-      unitSpec
-      integrationSpec conn
+      describe "Database.MySQL.Simple.unitSpec" unitSpec
+      describe "Database.MySQL.Simple.integrationSpec" $ integrationSpec conn
```

Comment:
changed up the hspec formatting a bit

Reply:
Sorry, I've just introduced a conflict here - maybe rebase, if you would.